CLAUDE.md

```diff
@@ -0,0 +1,25 @@
+# CLAUDE.md
```

Comment:
I have mixed feelings about checking this in to encourage more AI contributions.